apps/masterbots.ai/app/(browse)/[category]/[threadId]/page.tsx

```
@@ -4,7 +4,7 @@ import { ChatPageProps } from '@/app/c/[category]/[chatbot]/page'
 
 export { generateMbMetadata as generateMetadata } from '@/lib/metadata'
 
-export default async function ChatPage({ params }: ChatPageProps) {
+export default async function ChatPage({ params }: any) {
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)